app/assets/stylesheets/dashboard.scss

```diff
@@ -313,6 +313,20 @@ p.debate-status {
       height: 60px;
     }
 
+    .left .image {
```

Comment:
Nazwa klasy kontenera jest spowodowana wykorzystaniem frameworka sass-rails. Jest to klasa ustawiająca element po lewej stronie. Mogę zmienić nazwę i ustawić float na left w naszym pliku css, tylko czy jest sens to robić?